src/App.vue

```diff
@@ -823,6 +833,23 @@ export default {
         this.nameRequestInvalidDialog = false
         this.fetchData()
       }
+    },
+
+    /** Request and Download Business Summary Document. */
+    async downloadBusinessSummary (): Promise<void> {
+      this.showSpinner = true
+      const summaryDocument: DocumentIF = {
+        title: 'Summary',
+        filename: `${this.businessId} - Summary - ${this.getCurrentDate}.pdf`,
```

Comment:
Remove hyphen between Business ID and title.

Examples:
CP1002494 Certificate - 2022-01-13.pdf
CP1002494 Receipt - 2022-01-13.pdf

Reply:
ty, will do!